modules/olm-checking-deleted-operators.adoc

```diff
@@ -0,0 +1,156 @@
+
+* Go to OperatorHub in the namespace. It will show the Operator as not installed, since it is looking for the Operator’s subscription and not its listing under Operator Management. So the Operator is still partially installed.
+
+If you try to install the Operator again, it would not add a new entry to
+the Operator Management screen since the Operator is still there. But the
+bits under the Operator are not being updated. If you want to remove all
+traces of the uninstalled Operator from your cluster, you can do that
+from the command line.
+
+== Removing all traces of a deleted Operator
+
+Using the *oc* command, you can look for and clean out traces of deleted
+Operators. While logged into your cluster as a cluster administrator, do
+the following:
+
+.Procedure
+
+. To look for traces of the uninstalled Operator, list the catalogsource
+for the openshift-operators namespace:
++
+....
+$ oc get catalogsource -n openshift-operators
+NAME                                    NAME                TYPE PUBLISHER AGE
+installed-community-openshift-operators Community Operators grpc Community 10m
+....
++
+This shows the name of the pod containing the information for the community
+operators that are installed in the openshift-operators namespace:
++
+....
+installed-community-openshift-operators
+....
+
+. To see that the pod is running, type:
++
+....
+$ oc get pods -n openshift-marketplace | grep installed
+NAME                                          READY STATUS  RESTARTS AGE
+installed-community-openshift-operators-7b... 1/1   Running 0       165m
+....
+
+. By listing the catalogsourceconfig for installed-community-openshift-operators,
+you can see any Operators that were ever installed in this namespace, even if
+they were since deleted. For example, even though we had installed and deleted
+jaeger, this command shows jaeger as being still listed:
++
+....
+$ oc get catalogsourceconfig -n openshift-marketplace \
+    installed-community-openshift-operators -o yaml
+    ...
+        kind: CatalogSourceConfig
+        ...
+        spec:
+          csDisplayName: Community Operators
+          csPublisher: Community
+          packages: jaeger
+          targetnamespace: openshift-operators
+         ...
+        packageMap:
+          jaeger:  0.0.5
+....
+
+The previous output shows that the jaeger package is still in the
+CatalogSourceConfig, despite having been deleted (unsubscribed) by OLM.
+To clean that out you can either remove the one Operator or delete the
+entire CatalogSourceConfig.
+
+== Removing a single Operator from the CatalogSourceConfig
+
+To remove a single deleted Operator from the CatalogSourceConfig,
+you can remove its package name from the list in the CSC. To do
+this, you need to copy the CatalogSourceConfig, delete the
```

Comment:
You can alternatively edit the existing CSC using "oc edit"

Reply:
Yes. I will do that and use it as part of a new section on deleting an Operator from the command line.

Reply:
@SamiSousa I added a new section on deleting operators from the CLI and replaced the steps on deleting and changing the CatalogSourceConfig to use your "oc edit" suggestion. Take a look and let me know if other changes are needed.